### PR TITLE
Feature/forced exit

### PR DESCRIPTION
**Pull request by EvgenKor**
Added `forced-exit` CLI command to guide through the emergency exit process

image

The beta version of console was deployed at https://console.thgkjlr.website/
Current pool which support emergency exit: `BOB2USDC-goerli`

Associated PRs:
- https://github.com/zkBob/libzkbob-rs/pull/74
- https://github.com/zkBob/zkbob-client-js/pull/166

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -15,16 +15,19 @@ var pjson = require('../package.json');
 
 const bs58 = require('bs58');
 
+function account(_this: any): Account {
```
Comment (AllFi) [inline src/commands.ts:18]: What is the reason for replacing `this.account` with `account(this)`? I am just curious.

**Review by AllFi — APPROVED** [review]
Looks good to me

File: src/commands.ts (hunk 1 above)
Comment (EvgenKor) [reply src/commands.ts:18]: It's just a jQuery peculiarity. `this` pointer passed to the function which called directly from the terminal (e.g. `transferToken(to: string, amount: string)` or `depositShielded(amount: string, times: string)`). But in the internal subroutines (like `prinfForcedExit(_this: any, fe: any): Promise<void>`) `this` pointer becomes `undefined` and we need find a way to get `account` object. So this ugly crutch made for that :)

Comment (AllFi) [reply src/commands.ts:18]: Got it, thank you for the explanation.

**Pull request merged**